src/flows/shared/steps/OrganizeCollection/Sidebar/Sidebar.tsx

```diff
@@ -63,6 +78,20 @@ function Sidebar({ tokensRef, sidebarTokens, queryRef }: Props) {
   const nonNullTokens = removeNullValues(allTokens);
   const sidebarTokensAsArray = useMemo(() => convertObjectToArray(sidebarTokens), [sidebarTokens]);
 
+  // Only show blank space + add account button
+  // 1. if the user don't have tezos account
+  // 2. the selected chain is tezos
+  // In future, we might enabled this on other chain too
+  const isTezosAccountConnected = useMemo(() => {
+    if (selectedChain !== 'Tezos') return true;
+
+    const isUserHasTezosWallet = query.viewer?.user?.wallets?.some(
+      (wallet) => wallet?.chainAddress?.chain === 'Tezos'
+    );
+
+    return isUserHasTezosWallet ?? false;
+  }, [query, selectedChain]);
```

Comment:
sorry for not clarifying this earlier, i also think we should enable this for ETH, because in theory someone could be connected with tezos wallet but _not_ with ETH. in this scenario we should probably produce the same behavior. 

POAP is an edge case, because it's tied to ETH.

so we could do something like this:
```js
const ownsWalletFromSelectedChain = useMemo(() => {
    let chain = selectedChain;
    if (selectedChain === 'POAP') {
        chain = 'Ethereum'
    }

    const ownsWalletFromChain = query.viewer?.user?.wallets?.some(
        (wallet) => wallet?.chainAddress?.chain === selectedChain
    );

    return ownsWalletFromChain ?? false
}, [query, selectedChain]);
```

Reply:
this has some implications for the rest of the logic, which i've commented below

Reply:
Don't feel sorry @Robinnnnn! It make totally sense to apply on all chains. 

Thanks for the guide! I've updated it